litmus-portal/frontend/src/components/Header/ProfileDropDown.tsx

```diff
@@ -0,0 +1,134 @@
+
+  // Get username from JWT
+  const username = getUsername();
+
+  // Get the userEmail from JWT
+  const userEmailToken = getUserEmail();
+
+  const projectID = getProjectID();
+  const projectRole = getProjectRole();
+
+  const [userEmail, setuserEmail] = useState<string>(userEmailToken);
+
+  // Run query to get the data in case it is not present in the JWT
+  useQuery<CurrentUserDetails>(GET_USER_INFO, {
+    skip: userEmail !== undefined,
+    variables: { username },
+    onCompleted: (data) => {
+      setuserEmail(data.getUser.email);
+    },
+  });
+
+  // Handle clicks
+  const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  const open = Boolean(anchorEl);
+  const id = open ? 'headerProfileDropdown' : undefined;
+
+  const initials = userInitials(username);
+
+  return (
+    <div className={classes.profileDropdown} data-cy="headerProfileDropdown">
+      <IconButton edge="end" onClick={handleClick}>
+        <Avatar>{initials}</Avatar>
+      </IconButton>
+      <Popover
+        id={id}
+        open={open}
+        anchorEl={anchorEl}
+        onClose={handleClose}
+        anchorOrigin={{
+          vertical: 'bottom',
+          horizontal: 'left',
+        }}
+        transformOrigin={{
+          vertical: 'top',
+          horizontal: 'left',
+        }}
+      >
+        <div className={classes.profileDropdownPopover}>
+          <Typography>{t('header.profileDropdown.signedIn')}</Typography>
+          {userEmail ? (
+            <>
+              <Typography className={classes.profileSet}>
+                {userEmail}
+              </Typography>
+            </>
+          ) : (
+            <div
+              className={`${classes.profileDropdownRow} ${classes.profileUnset}`}
+            >
+              <Typography id="emailUnset">
+                {t('header.profileDropdown.emailUnset')}
+              </Typography>
+              <Link
+                to={{
+                  pathname: '/settings',
+                  search: `?projectID=${projectID}&projectRole=${projectRole}`,
+                }}
+                onClick={() => tabs.changeSettingsTabs(0)}
+              >
+                <Typography title="Go to settings">
+                  {t('header.profileDropdown.emailSet')}
+                </Typography>
+              </Link>
+            </div>
+          )}
+          <div
+            className={`${classes.profileDropdownRow} ${classes.profileButtons}`}
+          >
+            <ButtonFilled
+              title="Logout from the portal"
+              onClick={() => logout()}
+            >
+              {t('header.profileDropdown.logout')}
+              <img id="logoutIcon" src="./icons/logout.svg" alt="logout" />
+            </ButtonFilled>
+            <ButtonOutlined
+              title="Edit your profile"
```

Comment:
@vanshBhatia-A4k9 can we use translation here if possible?

Reply:
Yes @ishangupta-ds , will be adding them all as anyways this just first of probably 2-3 PRs on this same feature. Requesting you to please ignore these for now as I've already crossed 100 file changes.

Reply:
This one is a completely new file also translation file already has some changes in your PR specific to this page, I don't think it will increase file changes on adding these two text lines. @vanshBhatia-A4k9

Reply:
If you meant for only this one then cool, changing across all the files I touched would be an overhead. Thanks :+1:

Reply:
No, as the header component is completely refactored and is practically implemented from scratch it would be good to push it complete with all translations. Just this file change will be good.